### PR TITLE
Fix for https://github.com/ibc/libsdptransform/issues/16

### PR DESCRIPTION
**Pull request by christophe-calmejane**
Adding support for parsing unsigned values
SessionID and SessionVersion now parsed as unsigned value

### CODE REVIEW

File: src/parser.cpp [hunk 1]
```diff
@@ -293,15 +294,28 @@ namespace sdptransform
...
 				std::istringstream iss(str);
-				long long ll;
+				std::int64_t ll;
 
 				iss >> std::noskipws >> ll;
 
 				if (iss.eof() && !iss.fail())
-					return std::stoll(str);
+					return ll;
 				else
```
Comment (ibc) [inline src/parser.cpp:319]: Why this change?

Comment (christophe-calmejane) [reply src/parser.cpp:319]: While I do agree this change is unrelated to the fix this PR solves, there is no need to convert again the string which was already converted to integer by the stringstream.

Comment (ibc) [reply src/parser.cpp:319]: Oh true.

**Comment by ibc** [comment]
Merging, thanks a lot.
